crates/transaction-pool/src/error.rs

```diff
@@ -232,6 +232,10 @@ pub enum InvalidPoolTransactionError {
     /// invocation.
     #[error("intrinsic gas too low")]
     IntrinsicGasTooLow,
+    /// Op-supervisor deemed transaction invalid
+    // TODO: we should add proper error and include all option in here
+    #[error("invalid cross tx")]
+    InvalidCrossTx,
```

Comment:
this should be a standalone 

https://github.com/paradigmxyz/reth/blob/03c5abba887452a00680ab5e8a1aee34811158a1/crates/transaction-pool/src/error.rs#L16-L16

error type instead, which is used here:

https://github.com/paradigmxyz/reth/blob/03c5abba887452a00680ab5e8a1aee34811158a1/crates/transaction-pool/src/error.rs#L228-L230